src/components/seller-form/seller-form.tpl.html

```diff
@@ -7,50 +7,110 @@
           <div class="row">
             <div class="col-sm-12">
               <div class="panel">
-                <form>
-                  <div class="form-group">
-                    <label for="sellerName">Seller Name</label>
-                    <input type="text" class="form-control" placeholder="Seller Name">
-                  </div>
-                  <div class="form-group">
-                    <label for="Countries">Countries</label>
-                    <div class="dropdown">
-                      <button class="btn btn-secondary dropdown-toggle" type="button" 
-                      id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        Dropdown button
-                      </button>
-                      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                        <span>UK</span>
-                        <span>USA</span>
-                        <span>USA</span>
-                        <span>FR</span>
+                <form ng-submit="$ctrl.submit()" name="sellerDetail">
+                  <!-- <h1>Seller Form</h1> -->
+                    <div class="form-group">
+                        <label class="col-sm-2 col-form-label">Seller Name*</label>  
+                        <div class="col-sm-10">
+                            <input ref="sellerName" type="text" class="form-control" id="sellerName" ng-model="$ctrl.sellerName"/>
+                        </div>
+                    </div>
+                    
+                    <div class="form-group">
+                        <label class="col-sm-2 col-form-label">Currencies*</label>  
+                        <div class="col-sm-10">
+                            <select id="currencies" class="form-control" name="currencies" ng-model="$ctrl.currencies">
+                                <option value="USD">USD</option>
+                                <option value="GBP">GBP</option>
+                                <option value="INR">INR</option>
+                                <option value="RMB">RMB</option>
+                            </select>
+                        </div>
+                    </div>
+              
+                    <div class="form-group">
+                        <label class="col-sm-2 col-form-label">Offices*</label>
+                        <div class="col-sm-10">
+                            <select id="offices" class="form-control" name="officeval" ng-model="$ctrl.office">
+                                <option value="volvo">CA</option>
```

Comment:
[Medium]: This snippet looks copied, please make sure to update correct value in the attribute